### PR TITLE
Match dates format with API

### PR DESCRIPTION
**Pull request by TheAnkurGoswami**
## What does this PR do?
Fixes timestamp format.

## Changes

* Instead of using microsecond in the timestamp, now we use millisecond (like the api).

## Before reviewing

- [x] Did you test this PR?
- [ ] Is there any migration or seeding necessary?
- [ ] If a new field is added to signals in the api, did you update the sample test files?

### Links:

- Jira: 
  - https://myaccern.atlassian.net/browse/RSL-665
  - https://myaccern.atlassian.net/browse/RSL-669

### CODE REVIEW

File: packages/python/accern_data/accern_data.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
     has_iprogress,
     is_example_url,
     MessageIndicator,
+    mirco_to_millisecond,
```
Comment (shaunak-gupta) [inline packages/python/accern_data/accern_data.py:37]:
```suggestion
    micro_to_millisecond,
```
typo - fix in all places

File: packages/python/accern_data/accern_data.py [hunk 2]
```diff
@@ -315,12 +326,22 @@ def finish_day(
             merge_csv_file(fname)
             self._cols = None
 
+    def stringify_dates(self, obj: pd.DataFrame) -> pd.DataFrame:
+
+        def micro_to_milli(timestamp: pd.Timestamp) -> str:
+            return micro_to_millisecond(
+                    pd.to_datetime(timestamp).strftime(DATETIME_FORMAT))
```
Comment (shaunak-gupta) [inline packages/python/accern_data/accern_data.py:333]: the indentation seems off here

File: packages/python/accern_data/accern_data.py [hunk 3]
```diff
@@ -287,6 +291,13 @@ def _write_cols(self) -> None:
     def add_result(self, signal: pd.DataFrame) -> None:
         fname = self.get_path(is_by_day=self._is_by_day)
         tmp_fname = get_tmp_file_name(fname)
+
+        def micro_to_milli(timestamp: pd.Timestamp) -> str:
+            return micro_to_millisecond(
+                pd.to_datetime(timestamp).strftime(DATETIME_FORMAT))
+
+        for col in ["harvested_at", "crawled_at", "published_at"]:
+            signal.loc[:, col] = signal.loc[:, col].apply(micro_to_milli)
```
Comment (shaunak-gupta) [inline packages/python/accern_data/accern_data.py:300]: weird that the SettingWithCopyWarning doesn't appear here

**Review by shaunak-gupta — COMMENTED** [review]
minor changes

File: packages/python/accern_data/accern_data.py (hunk 3 above)
Comment (TheAnkurGoswami) [reply packages/python/accern_data/accern_data.py]: that should be removed. idk why i kept this

**Review by shaunak-gupta — APPROVED** [review]
lgtm

**Review by shaunak-gupta — APPROVED** [review]

**Pull request merged**